File: Modul-2/Soal 5/PRAK205-2410817210014-ZAVIER GHANI.py
def hitung_segitiga(tinggi, sisi_miring):
    alas = (sisi_miring ** 2 - tinggi ** 2) ** 0.5
    keliling = tinggi + sisi_miring + alas
    luas = 0.5 * alas * tinggi

    print(f"Alas = {alas:.0f} cm")
    print(f"Tinggi = {tinggi:.0f} cm")
    print(f"Keliling = {keliling:.0f} cm")
    print(f"Luas = {luas:.0f} cm^2")

# Test case 1
print("Test Case 1")
A1, B1 = map(float, input("Masukkan tinggi (A) dan sisi miring (B) (dipisah spasi): ").split())
hitung_segitiga(A1, B1)

print("\n") 

# Test case 2
print("Test Case 2")
A2, B2 = map(float, input("Masukkan tinggi (A) dan sisi miring (B) (dipisah spasi): ").split())
hitung_segitiga(A2, B2)

File: Modul-2/Soal 5/PRAK205-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    float A, B;

    // Test case 1
    printf("Test Case 1\n");
    printf("Masukkan tinggi (A) dan sisi miring (B) (dipisah spasi): ");
    scanf("%f %f", &A, &B);

    float alas = sqrt(B * B - A * A);
    float keliling = A + B + alas;
    float luas = 0.5 * alas * A;

    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", A);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    printf("\n"); 

    // Test case 2
    printf("Test Case 2\n");
    printf("Masukkan tinggi (A) dan sisi miring (B) (dipisah spasi): ");
    scanf("%f %f", &A, &B);

    alas = sqrt(B * B - A * A);
    keliling = A + B + alas;
    luas = 0.5 * alas * A;

    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", A);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}
